ci_framework/roles/repo_setup/tasks/artifacts.yml

```diff
@@ -1,15 +1,17 @@
 ---
-- name: Run repo-setup=get-hash
-  ansible.builtin.shell: |
-    {{ cifmw_repo_setup_basedir }}/venv/bin/repo-setup-get-hash \
-      --dlrn-url {{ cifmw_repo_setup_dlrn_uri[:-1] }} \
-      --os-version {{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }} \
-      --tag {{cifmw_repo_setup_promotion }} \
-      --release {{ cifmw_repo_setup_branch }} \
-    {% if cifmw_repo_setup_component is defined %}
-      --component {{ cifmw_repo_setup_component }} \
-    {% endif %}
+- name: Run repo-setup-get-hash
+  ansible.builtin.command:
+    cmd: >-
+      {{ cifmw_repo_setup_basedir }}/venv/bin/repo-setup-get-hash
+      --dlrn-url {{ cifmw_repo_setup_dlrn_uri[:-1] }}
+      --os-version {{ cifmw_repo_setup_os_release }}{{ cifmw_repo_setup_dist_major_version }}
+      --tag {{cifmw_repo_setup_promotion }}
+      --release {{ cifmw_repo_setup_branch }}
+      {% if cifmw_repo_setup_component is defined %}
+      --component {{ cifmw_repo_setup_component }}
+      {% endif %}
       --json
+  environment: "{{ cifmw_repo_setup_env | default ({}) }}"
```

Comment:
We already have defaults {} in defaults/main.yaml so | default ({}) is redundant so okay to keep it.